pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -625,7 +625,7 @@ func (o *InstallOptions) writePod(rawPodBytes []byte, manifestFileName, resource
 func writeConfig(content []byte, fullFilename string) error {
 	klog.Infof("Writing config file %q ...", fullFilename)
 
-	filePerms := os.FileMode(0644)
+	filePerms := os.FileMode(0600)
 	if strings.HasSuffix(fullFilename, ".sh") {
 		filePerms = 0755
```

Comment:
Do we also want to change this to 700? If so, should we just use `writeSecret` instead?

Reply:
@hasbro17 I found `cluster-backup.sh` and `cluster-restore.sh` scripts getting copied from a CM. It seems save to reduce the permissions to just 0700. Though, better to double check. Do you know if both scripts can work with the limited permissions?